### DISCUSSION TITLE
Add ability to configure Home furniture/sections

### DISCUSSION THREAD

**Discussion by jonaballen** [gotson/komga#1539] [Ideas]
### Describe your suggested feature

**Configure furniture on Home (Nice to have)**
It would be nice to be able to be able to select which sections that is available in the Home section.
Afaik, there are 

- Keep reading
- On Deck
- Recently Released Books
- Recently Added Books
- Recently Added Series
- Recently Updated Series
- Recently Read Books

The list has been growing and adds vertical space, especially on mobile screens.
As a user, it would be convenient to be able to decide exactly which sections I want to include, to minimize clutter.
This could also be extended to each Library > Recommended page

P.s. i only found #931 but its a different request, adding ability to hide items added to specific collections, the above suggestion is more general option to select furniture on the Home page

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
This will be included in the next release

**Reply by gotson** [reply]
🎉 This issue has been resolved in `1.20.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.20.0))
